portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,11 +50,19 @@ public void init() {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+    
+
       //Get the input from the form.
       String comment = request.getParameter("textInput");
 
-      //Add input to messages array.
-      messages.add(comment);
+      //Create entity
```

Comment:
As Tamas mentioned in the earlier pull-request, add a space after "//"

Reply:
Done!